### PR TITLE
[Woptim] rsc rajaperf mpiexec flag

### PR DESCRIPTION
**Pull request by adrienbernede**
### Summary

This branch hosts a Fix attempt from RADIUSS Spack Configs for RAJAPerf CI.

### CODE REVIEW

File: scripts/gitlab/build_and_test.sh [hunk 1]
```diff
@@ -164,9 +164,13 @@ then
...
     if ! $cmake_exe --build . -j ${core_counts[$truehostname]}
     then
-        echo "ERROR: compilation failed, building with verbose output..."
+        echo "[Error]: compilation failed, building with verbose output..."
+        echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+        echo "~~~~~ Running make VERBOSE=1"
+        echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
         $cmake_exe --build . --verbose -j 1
     else
+        # todo this should use cmake --install once we use CMake 3.15+ everywhere
```
Comment (rhornung67) [inline scripts/gitlab/build_and_test.sh:173]: We are using CMake 3.20 or newer in all GItLab CI jobs on all LC machines

**Review by rhornung67 — COMMENTED** [review]
CMake version comment in GitLab build-and-test script

**Comment by adrienbernede** [comment]
@rhornung67 This is ready. Once merged, we can point https://github.com/LLNL/RAJAPerf/pull/332 at the corresponding commit in raja@develop.

**Review by MrBurmark — APPROVED** [review]

**Comment by rhornung67** [comment]
@adrienbernede RAJA Perf develop is now pointing at RAJA develop after I merged https://github.com/LLNL/RAJAPerf/pull/405

**Pull request merged**